tools/precommit.py

```diff
@@ -113,6 +113,42 @@ def setup_tizen_root(tizen_root):
             'https://github.com/pmarcinkiew/tizen3.0_rootstrap.git',
             tizen_root])
 
+def copy_tiznert_stuff(tizenrt_root, iotjs_dir):
+    tizenrt_iotjsapp_dir = fs.join(tizenrt_root, 'apps/system/iotjs')
+    if not fs.exists(tizenrt_iotjsapp_dir):
+        iotjs_tizenrt_appdir = fs.join(iotjs_dir,
+                                       'config/tizenrt/artik05x/app')
+        ex.check_run_cmd('cp',
+                         ['-r', iotjs_tizenrt_appdir, tizenrt_iotjsapp_dir])
+
+    tizenrt_config_dir = fs.join(tizenrt_root, 'build/configs/artik053/iotjs')
+    if not fs.exists(tizenrt_config_dir):
+        iotjs_config_dir = \
+            fs.join(iotjs_dir, 'config/tizenrt/artik05x/configs')
+        ex.check_run_cmd('cp',
+                         ['-r', iotjs_config_dir, tizenrt_config_dir])
+
+def setup_tizenrt_root(tizenrt_root):
+    if fs.exists(tizenrt_root):
+        fs.chdir(tizenrt_root)
+        ex.check_run_cmd('git', ['pull'])
+        fs.chdir(path.PROJECT_ROOT)
+    else:
+        ex.check_run_cmd('git', ['clone',
+            'https://github.com/Samsung/TizenRT.git',
+            tizenrt_root])
+
+    copy_tiznert_stuff(tizenrt_root, path.PROJECT_ROOT)
+    tizenrt_tools = fs.join(tizenrt_root, 'os/tools')
+    fs.chdir(tizenrt_tools)
+    ex.check_run_cmd('./configure.sh', ['artik053/iotjs'])
+    fs.chdir('..')
+    ex.check_run_cmd('make', ['context'])
+
+def build_tizenrt(tizenrt_root, iotjs_rootdir):
+    fs.chdir(fs.join(tizenrt_root, 'os'))
+    rflag = 'R=1' if (buildtype == "release") else 'R=0'
+    ex.check_run_cmd('make', ['IOTJS_ROOT_DIR=' + iotjs_rootdir, rflag])
```

Comment:
Done:
 - passed IOTJS_LIB_DIR explicitly,
 - stopped passing buildtype to TizenRT build, since it does not work anyway (see: https://github.com/Samsung/TizenRT/issues/290).